includes/classes/class-welcome-page.php

```diff
@@ -205,4 +205,184 @@ public static function render_summary() {
+							<div>' . __( 'Join our email list and get event reminders and recordings in your inbox.', 'accessibility-checker' ) . '</div>
+							<div>
+								<div id="Zc_SignupSuccess" style="display: none;"></div>
+							</div>
+							
+							<form method="POST" id="zcampaignOptinForm" style="margin: 0px; width: 100%" action="https://zmp-glf.maillist-manage.com/weboptin.zc" target="_zcSignup">
+							
+							<div class="SIGNUP_FLD edac-mt-3" id="edac-opt-in-email">
+								<label style="font-weight: 600;" for="EMBED_FORM_EMAIL_LABEL">Email Address (Required)</label>
+								<input style="max-width: 100%;" type="email" changeitem="SIGNUP_FORM_FIELD" name="CONTACT_EMAIL" id="EMBED_FORM_EMAIL_LABEL" aria-describedby="email-info-region" value="' . esc_attr( $email ) . '">
+								<div class="edac-mt-3" id="email-info-region" class="info-message">
+									' . esc_html( __( 'Note: The Email Address field has been auto-populated with your WordPress email.', 'accessibility-checker' ) ) . '
+								</div>
+							</div>
+							<div class="edac-mt-0" style="display: none;" id="errorMsgDiv">' . __( 'Please enter a valid email address.', 'accessibility-checker' ) . '</div>
+						
+							<div class="SIGNUP_FLD edac-mt-3">
+								<input type="button" class="button" name="SIGNUP_SUBMIT_BUTTON" id="zcWebOptin" value="' . __( 'Subscribe', 'accessibility-checker' ) . '">
+							</div>
+						
+							
+							<input type="hidden" id="fieldBorder" value="">
+							<input type="hidden" id="submitType" name="submitType" value="optinCustomView">
+							<input type="hidden" id="emailReportId" name="emailReportId" value="">
+							<input type="hidden" id="formType" name="formType" value="QuickForm">
+							<input type="hidden" name="zcvers" value="3.0">
+							<input type="hidden" name="oldListIds" id="allCheckedListIds" value="">
+							<input type="hidden" id="mode" name="mode" value="OptinCreateView">
+							<input type="hidden" id="document_domain" value="">
+							<input type="hidden" id="zc_Url" value="zmp-glf.maillist-manage.com">
+							<input type="hidden" id="new_optin_response_in" value="2">
+							<input type="hidden" id="duplicate_optin_response_in" value="2">
+							<input type="hidden" name="zc_trackCode" id="zc_trackCode" value="ZCFORMVIEW">
+							<input type="hidden" id="viewFrom" value="URL_ACTION">
+							<input type="hidden" name="zx" id="cmpZuid" value="' . esc_attr( $zx ) . '">
+							<input type="hidden" id="zcld" name="zcld" value="' . esc_attr( $zcld ) . '">
+							<input type="hidden" id="zctd" name="zctd" value="' . esc_attr( $zctd ) . '">
+							<input type="hidden" id="zc_formIx" name="zc_formIx" value="' . esc_attr( $zc_form_ix ) . '">
+							<span style="display: none" id="dt_CONTACT_EMAIL">1,true,6,Contact Email,2</span>
+							</form>
+
+							<!-- Confirmation Message -->
+							<div id="confirmationMessage" tabindex="-1" role="alert" style="outline: none;">
```

Comment:
The confirmation message is being read out twice by the screen reader. Remove role="alert" and test that it's only being read out once.